settings.gradle

```diff
@@ -91,6 +91,7 @@ include ':instrumentation:hibernate:hibernate-4.0'
 include ':instrumentation:hibernate:hibernate-4.3'
 include ':instrumentation:hibernate:hibernate-common'
 include ':instrumentation:http-url-connection'
+include ':instrumentation:httpclient'
```

Comment:
i think this name is too generic, since there are many httpclient instrumentations, i think my suggestion would be `java-httpclient`

Reply:
renamed